lib/ret_web/channels/hub_channel.ex

```diff
@@ -152,7 +152,11 @@ defmodule RetWeb.HubChannel do
   def handle_in("events:end_streaming", _payload, socket), do: socket |> set_presence_flag(:streaming, false)
 
   # Captures all inbound NAF messages that result in spawned objects.
-  def handle_in("naf" = event, %{"data" => %{"isFirstSync" => true, "template" => template}} = payload, socket) do
+  def handle_in(
+        "naf" = event,
+        %{"data" => %{"isFirstSync" => true, "persistent" => false, "template" => template}} = payload,
+        socket
+      ) do
```

Comment:
since we're no longer dealing with persistent entities here, I'd suggest we re-rename the `created_entities` map to `secured_entities` since the persistent entities will no longer be registered